pyvista/core/pointset.py

```diff
@@ -415,19 +416,71 @@ def is_all_triangles(self):
 
         """
         # Need to make sure there are only face cells and no lines/verts
-        faces = self.faces  # grab once as this takes time to build
-        if not len(faces) or len(self.lines) > 0 or len(self.verts) > 0:
+        if not self.n_faces or self.n_lines or self.n_verts:
             return False
 
-        # All we have are faces, check if all faces are indeed triangles
-        if faces.size % 4 == 0:
-            return (faces[::4] == 3).all()
-        return False
+        # in VTK9, they use connectivity and offset rather than cell
+        # data.  Use the new API as this is faster
+        if _vtk.VTK9 and False:
```

Comment:
Is this False check intended?  How can this ever execute?

Reply:
This was for timing testing and I forgot to remove it.  On mobile and will remove it at home.  Thanks for finding this.